internal/cli/organizations/list_test.go

```diff
@@ -22,28 +22,83 @@ import (
 	"github.com/golang/mock/gomock"
 	"github.com/mongodb/mongodb-atlas-cli/atlascli/internal/flag"
 	"github.com/mongodb/mongodb-atlas-cli/atlascli/internal/mocks"
+	"github.com/mongodb/mongodb-atlas-cli/atlascli/internal/pointer"
 	"github.com/mongodb/mongodb-atlas-cli/atlascli/internal/test"
 	atlasv2 "go.mongodb.org/atlas-sdk/v20231115014/admin"
 )
 
 func TestList_Run(t *testing.T) {
-	ctrl := gomock.NewController(t)
-	mockStore := mocks.NewMockOrganizationLister(ctrl)
+	tests := []struct {
+		name      string
+		expected  *atlasv2.PaginatedOrganization
+		returnErr error
+	}{
+		{
+			name: "non-nil result",
+			expected: &atlasv2.PaginatedOrganization{
+				Results: &[]atlasv2.AtlasOrganization{{}, {}, {}},
+			},
+			returnErr: nil,
+		},
+		{
+			name: "nil result",
+			expected: &atlasv2.PaginatedOrganization{
+				Results: nil,
+			},
+			returnErr: nil,
+		},
+		{
+			name:      "no results",
+			expected:  &atlasv2.PaginatedOrganization{},
+			returnErr: nil,
+		},
+		{
+			name: "no results",
+			expected: &atlasv2.PaginatedOrganization{
+				Results: &[]atlasv2.AtlasOrganization{},
+			},
+			returnErr: nil,
+		},
+		{
+			name: "with results",
+			expected: &atlasv2.PaginatedOrganization{
+				Results: &[]atlasv2.AtlasOrganization{
+					{
+						Id:   pointer.Get("test"),
+						Name: "test",
+					},
+				},
+			},
+			returnErr: nil,
+		},
+	}
+
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			ctrl := gomock.NewController(t)
```

Comment:
can you parallelize this?
```suggestion
			t.Parallel()
			ctrl := gomock.NewController(t)
```